### PR TITLE
Redirect at login

### PR DESCRIPTION
**Pull request by demerzel3**

### CODE REVIEW

File: src/pages/Login.jsx [hunk 1]
```diff
@@ -1,9 +1,13 @@
...
-    <div>
-        <a href="https://github.com/login/oauth/authorize?client_id=Iv1.ae8d32938d68d499">Login with GitHub</a>
-    </div>
-);
+const Login = ({ location: { pathname, search, hash } }) => {
+    localStorage.setItem('redirect-to', [pathname, search, hash].join(''));
+
+    return (
+        <div>
+            <a href="https://github.com/login/oauth/authorize?client_id=Iv1.ae8d32938d68d499">Login with GitHub</a>
```
Comment (mattiaocchiuto) [inline src/pages/Login.jsx]: If we need to reuse the same client_id in other places I think we should move it in a constants.
Maybe the same could be for the _github basePath_

File: src/pages/Login.jsx [hunk 2]
```diff
@@ -1,9 +1,13 @@
 import React from 'react';
 
-const Login = () => (
-    <div>
-        <a href="https://github.com/login/oauth/authorize?client_id=Iv1.ae8d32938d68d499">Login with GitHub</a>
-    </div>
-);
+const Login = ({ location: { pathname, search, hash } }) => {
+    localStorage.setItem('redirect-to', [pathname, search, hash].join(''));
```
Comment (mattiaocchiuto) [inline src/pages/Login.jsx]: We should use a constant for every _localStorage Item_ in my opinion.

File: src/pages/Login.jsx (hunk 1 above)
Comment (demerzel3) [reply src/pages/Login.jsx]: The *right thing to do*™️ should be to turn the login URL to a prop and pass it from outside, from a constant or possibly a configuration file of some kind. But I think we can live with the static string for a while.

**Comment by demerzel3** [comment]
I moved the `localStorage` keys to a file of constants, I won't do the github base or client id here (though I agree, see my comment above) since they are out of the scope of this PR.

**Pull request merged**